docs/api/migration-guides/qiskit-1.0.mdx

```diff
@@ -0,0 +1,594 @@
+---
+title: Qiskit 1.0 Migration Guide
+description: Adapt to changes in Qiskit 1.0
+---
+
+# Qiskit 1.0 Migration Guide
+
+
+## Packaging changes
+
+Qiskit 1.0 uses a different packaging structure to previous Qiskits, and may cause problems in environments using packages that are not ready for Qiskit 1.0.
+
+<Admonition type="caution">
+We **do not** recommend attempting to upgrade an existing Python virtual environment from Qiskit 0.45 or before to Qiskit 1.0 in place.
+</Admonition>
+
+
+### For users
+
+You must start a new virtual environment to install Qiskit 1.0.
+It is very tricky and error-prone to upgrade an existing installation in-place to Qiskit 1.0.
+
+<Admonition type="info">
+This section will use the `venv` module that is part of the Python standard library in its examples.
+If you use a different tool (such as `virtualenv` or `conda`), you should consult its documentation if you are unsure of how the commands line up.
+
+The examples will assume a `bash`-like shell for non-Python command-line inputs.
+</Admonition>
+
+<span id="creating-a-venv"></span>
+#### Creating the new environment
+
+1. Create a new virtual environment, using your preferred (supported!) version of Python 3:
```

Comment:
Maybe insert a reference to the platform support section?

Reply:
I changed this to "Python 3.8+", and omitted the bit about actual OSes - I think a migration guide kind of assumes that you're migrating from something that's working, and we haven't changed the support matrix.